tutorials/2_sprite_animations_gestures/README.md

```diff
@@ -0,0 +1,241 @@
+# Basic: Sprites, Animations and Gestures
+
+This tutorial will introduce you to:
+
+ - `Sprite`: Sprites are how we draw images, or portions of an image in Flame.
+ - `SpriteAnimation`: SpriteAnimations are animations composed from sprites, where each sprite represents a frame.
+ - Gesture input: Input is a crucial part of game development, on this tutorial you will learn how to use one of the most basic of them, the tap detector.
+
+All the code of this tutorial code can be found [here](./code) and is based on the Flame `1.0.0-rc8` version.
+
+By the end of this tutorial you will have built a simple game which renders a button which when pressed, makes a small vampire robot run, that will look like this:
```

Comment:
```suggestion
By the end of this tutorial, you will have built a simple game which renders a button that, when pressed, makes a small vampire robot run. It will look like this:
```